src/main.cpp

```diff
@@ -55,6 +55,10 @@ int mspeed2, mspeed4 = MOTOR_MIN; // Motors along the y-axis
 //=> Changing their speeds will affect the roll
 
 
+/*
+*   Function to initialize the Gyro sample rate and give it the base offset value to give
+*   better calculations in the GetAngleMeasurements function
+*/
```

Comment:
Please align the '*' characters like seen elsewhere. Also, the spaces between the '*' characters and the first letter are way to long. Please, only one space.